etl/steps/data/garden/animal_welfare/2023-08-14/number_of_farmed_fish.meta.yml

```diff
@@ -0,0 +1,36 @@
+definitions:
+  common: |
```

Comment:
I would add `|-` here, because using it remove an additional line at the bottom of each description. There's no indentation, so it wouldn't affect anything else, I think

Reply:
Ah right, that solves the previous issue about the additional line :-)

Reply:
I think there is still one additional line, but in the case of the last indicator there was another one after doing this change.

Reply:
Where do you see the additional line? I don't see it either in the grapher step's variable descriptions or in [the sources tab](http://staging-site-prosado/admin/datasets/6174).

Reply:
Ah, sorry, forget about it. I saw it when I pushed it to my staging server, because I saw an additional line at the bottom of this file. I see it's not there now